ui/app/AppMain.qml

```diff
@@ -210,6 +210,11 @@ RowLayout {
             Layout.fillHeight: true
         }
     }
+
+    NotificationWindow {
+        id: notificationWindow
+        activeView: sLayout.currentIndex
```

Comment:
Did you forget to push the changes to `NotificationWindow`? I don't see where this new prop is used